addons/website/static/tests/tours/gray_color_palette.js

```diff
@@ -10,6 +13,11 @@ function waitForCSSReload() {
         // step.
         wTourUtils.goBackToBlocks(),
         wTourUtils.goToTheme(),
+        {
+            content: "Wait for no loading",
+            trigger: 'body:not(:has(.o_we_ui_loading)) iframe body:not(:has(.o_we_ui_loading))',
+            run: () => null,
+        },
```

Comment:
Well indeed, the extra_trigger part was probably what was indirectly "hiding" the issue. Adding a long enough delay for the next steps to work. Not sure it is worth restoring though: as the TODO I am adding indicates, we need a better way to consistenly wait for CSS reload. I'll find one once the current PR treating that are merged.

Meanwhile, I did not investigate further here since:
- Impossible to reproduce the issue (launch the test like 30 times, no fail)
- Saw an error in the extra_trigger of the problematic step according to runbot: just thought about fixing that :wink: 

Should have opened in draft, was planning to do a better commit message once multi-build confirmed this green :+1: Thanks for the reference, I'll add it in the commit message :+1: